### PR TITLE
Documentation improvements.

### PR DESCRIPTION
**Pull request by n1k0**
Mostly linking to appropriate sections of the detailed API docs.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
 Take the [tutorial](tutorial.md) to get you started, or start browsing the [API documentation](api.md).
 
+XXX https://doc.esdoc.org/github.com/Kinto/kinto.js/
```
Comment (Natim) [inline docs/index.md]: Why XXX ?

**Comment by Natim** [comment]
Great to be doing links between the two documentations.

**Comment by n1k0** [comment]
r=? @Natim

**Comment by Natim** [comment]
Apparently not: https://github.com/Kinto/kinto.js/blob/master/demo/demo.js#L3

**Comment by Natim** [comment]
It look good to me. r+

**Pull request merged**